src/main/java/net/sf/jabref/importer/fileformat/BibtexImporter.java

```diff
@@ -31,11 +31,11 @@
 public class BibtexImporter extends ImportFormat {
 
     /**
-     * @return false as that does not cause any harm in the current implementation of JabRef
+     * @return true as we have no effective way to decide whether a file is in bibtex format or not
      */
     @Override
     public boolean isRecognizedFormat(InputStream in) throws IOException {
-        return BibtexParser.isRecognizedFormat(new InputStreamReader(in));
+        return true;
```

Comment:
The check equaled more or less to a search of the `@` symbol. So it was purely semantic. Moreover, there is difference between a Bibtex and a Biblatex file on this semantic level. As I have explained above, in my opinion there is no easy way to tell if a file is a valid bibtex file or not. This is why I have deleted this method (which, by the way was only called upon importing a bibtex file, but not on opening one).